composer.json

```
@@ -17,7 +17,7 @@
         }
     ],
     "require": {
-        "php": "^7.4.0 || 8.0.*",
+        "php": "^7.4.0 || 8.0.* || 8.1.*",
```

Do you prefer instead
```suggestion
        "php": "^7.4.0 || ^8.0.0",
```